### PR TITLE
Updated ION to work with latest sidetree code

### PR DESCRIPTION
**Pull request by thehenrytsai**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "license": "Apache-2.0",
   "main": "dist/src/core.js",
   "dependencies": {
-    "@decentralized-identity/sidetree": "0.9.1",
+    "@decentralized-identity/sidetree": "../sidetree",
```
Comment (tplooker) [inline package.json:9]: @thehenrytsai if you are using an unreleased version of sidetree, why not use the unstable packages being pushed to npm on each commit?

Comment (tplooker) [reply package.json:9]: See https://www.npmjs.com/package/@decentralized-identity/sidetree/v/0.9.1-unstable.7887848 this is the latest release current to last commit on master for sidetree

Comment (tplooker) [reply package.json:9]: Could you please change this to `"@decentralized-identity/sidetree": "0.9.1-unstable.7887848",`

Comment (thehenrytsai) [reply package.json:9]: @tplooker, you are right, forgot that we have unstable releases we can reference. Will make an update.

**Pull request merged**